docs/dev/README.md

```diff
@@ -124,8 +124,8 @@ Make sure that the backend service running in the cluster is proxied using `kube
 
 **Note:** If you've exposed the backend by some other means than port-forwarding port 9097 as described above, set the `API_DOMAIN` environment variable to provide the correct details.
 
-e.g. to connect your local dev frontend to the Dashboard deployed on the robotcat cluster:
-`API_DOMAIN=https://dashboard.robotcat.tekton.dev/ npm start`
+e.g. to connect your local dev frontend to the Dashboard deployed on the robocat cluster:
```

Comment:
Poor robocat doesn't get an uppercase `R` and all the hard work its doing - I did check and its lower across project doc.

/approve
/lgtm

/meow sad